guardrails/validator_base.py

```diff
@@ -363,6 +495,26 @@ def to_runnable(self) -> Runnable:
 
         return ValidatorRunnable(self)
 
+    def _log_telemetry(self) -> None:
+        """Logs telemetry after the validator is called."""
 
-# Superseded by guardrails/types/validator.py::PydanticValidatorSpec
-ValidatorSpec = Union[Validator, Tuple[Union[Validator, str, Callable], str]]
+        if not self.kwargs.get("disable_tracer", False):
```

Comment:
This isn't being passed in anywhere.  This should come from the same config that we use to enable/disable in the Guard and the validator_service.

Reply:
This might need to happen sooner: https://github.com/guardrails-ai/guardrails/pull/815/files#r1659011429